tests/puppeteer/test/utils/globals.js

```diff
@@ -0,0 +1,7 @@
+global.URL_FO = process.env.URL_FO || 'http://localhost/prestashop/';
+global.URL_BO = process.env.URL_BO || URL_FO + 'admin-dev/';
+global.EMAIL = process.env.LOGIN || 'demo@prestashop.com';
+global.PASSWD = process.env.PASSWD || 'prestashop_demo';
+global.HEADLESS = process.env.HEADLESS || true;
+global.browser = null;
+global.page = null;
```

Comment:
I think that Global args could be used in linkchecker.js to.

Reply:
Are we sure we're gonna keep this folder architecture ? Because I need to require the `globals.js` file and it might move...

Reply:
I'm not so sure, We can wait until defining architecture is finished.

Reply:
I added it anyway, that's a good idea. We can always change it later, it's a relative path